### ISSUE TITLE
Add Dark Mode to UI 🌙

### ISSUE THREAD

**Issue by mohsinansari0705** [mohsinansari0705/Voice-Activated-Virtual-Assistant#8]
# Description
Implement a dark mode option for the virtual assistant’s UI.
## Tasks
- Add a toggle button for switching between light and dark themes.
- Ensure the UI elements are optimized for both modes.
- Skills Required: HTML, CSS, JavaScript
- Expected Outcome: Users can switch between light and dark modes for a better experience.

**Comment by Harshvardhan5703** [comment]
Hey, i want to work upon it...

**Comment by mohsinansari0705** [comment]
@Harshvardhan5703 Absolutely you can.
Before going to solve this issue I prefer you to complete #1 
Comment on the issue #1 for create the UI for the Assistant & after that you can do this.

**Comment by Harshvardhan5703** [comment]
Hey can you do it under hacktober

**Comment by mohsinansari0705** [comment]
Done, Now you can do it. @Harshvardhan5703

**Comment by Harshvardhan5703** [comment]
Bro where is the UI!!

**Comment by mohsinansari0705** [comment]
@Harshvardhan5703
Yup, there is no UI ever pulled by anyone but you can make UI for the assistant.
Go for the issue #1 .
